src/basket-service/models/basket-item.ts

```diff
@@ -138,14 +145,36 @@ export class BasketItem {
     return this.feeInShopperCurrency ? this.feeInShopperCurrency.value : null;
   }
 
+  getDeliveryFeeAmount () {
+    return this.deliveryFeeInShopperCurrency ? this.deliveryFeeInShopperCurrency.value : null;
+  }
+
+  getOriginalSalePriceAmountWithoutDeliveryFee () {
+    return this.getOriginalSalePriceAmount() - this.getDeliveryFeeAmount();
```

Comment:
I think you'll get an error if getDeliveryFeeAmount() is null here. Maybe the above function should default to 0?

Reply:
I tried to follow the pattern used by the other methods here - trying something like `8 - null` or `null - null` seem to work okay for me in the console!